envoy/common/exception.h

```diff
@@ -61,6 +61,13 @@ class EnvoyException : public std::runtime_error {
     return temp_status;                                                                            \
   }
 
+// This macro is used to return from a function directly if the status is not ok
+// without returning a status value.
+#define RETURN_ONLY_IF_NOT_OK_REF(variable)                                                        \
```

Comment:
I don't love this name but can't think of anything better =P

Reply:
I think this should be `RETURN_IF_NOT_OK_REF` but this name is used. orz

Maybe we can change others to `RETURN_STATUS_IF_NOT_OK_REF` and this one to `RETURN_IF_NOT_OK_REF`. If you think this naming is better.